themes/default/content/registry/packages/aws-static-website/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: AWS Static Website
+meta_desc: Provides an overview of the AWS Static Website component.
+layout: overview
+---
+
+This component makes it easy to deploy a static website to s3 along with an optional CloudFront distribution using any of the supported Pulumi programming languages including markup languages like YAML and JSON.
+
+## Examples
+
+{{< chooser language "typescript,go,python,yaml" >}}
+{{% choosable language typescript %}}
+
+```typescript
+import * as staticwebsite from "@pulumi/aws-static-website"
+
+const args =  {
+    sitePath: "../website/build",
+} as staticwebsite.WebsiteArgs
+
+const site = new staticwebsite.Website("website", args);
```

Comment:
We could make this even simpler and keep it Pulumi-idiomatic -- what do you think? (Also adds a line to export the URL.)

```suggestion
import * as staticwebsite from "@pulumi/aws-static-website"

const site = new staticwebsite.Website("website", {
    sitePath: "../website/build"
});

export const url = site.websiteURL;
```

Reply:
good point, thanks @cnunciato. that is MUCH better :slightly_smiling_face: